Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -155,10 +160,18 @@ class ActionOverlayTableViewHeader: UITableViewHeaderFooterView {
         let siteImageView = UIImageView()
         siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
         siteImageView.clipsToBounds = true
-        siteImageView.layer.cornerRadius = SimpleHighlightCellUX.CornerRadius
         return siteImageView
     }()
 
+    lazy var siteImageBackground: UIView = {
```

Comment:
If this was in a tableview with 100s of these in a table there might be an issue. But in this case I think you are okay 😄